Dockerfile.assisted-service-build

```diff
@@ -1,19 +1,23 @@
 FROM quay.io/edge-infrastructure/postgresql-12-centos7:latest
 
+COPY --from=quay.io/goswagger/swagger:v0.28.0 /usr/bin/swagger /usr/bin/goswagger
+COPY --from=quay.io/edge-infrastructure/swagger-codegen-cli:2.4.18 /opt/swagger-codegen-cli /opt/swagger-codegen-cli
```

Comment:
Why do we have our own version of `swagger-codegen-cli` ?

Reply:
we mainly needed something from quay.io to have less rate-limiters problems than on docker-hub